float64 robotX
float64 robotY
string direction
int64 time
string[] destNames
bool[] destDeliveryStatus
float64[] destXCoord
float64[] destYCoord
int64[] destTimeStart
int64[] destTimeEnd
